### DISCUSSION TITLE
Looking for an experienced plotter...Please help

### DISCUSSION THREAD

**Discussion by rdwarren13** [swar/Swar-Chia-Plot-Manager#1222] [Q&A]
Can you please help with what our set up should be with our current equipment? We are running an HP Workstation.

2 Intel Xeon CPU E5-2699 v4 @2.20Ghz
44 Cores, 88 Threads
256 GB RAM
2 8TB U.2 SSDs Model: MZWLJ7T6HALA-000V7
Thank you in advance for your help.

**Comment by feiyingCQ** [comment]
It's totally enough.
And you can try MadMax plotter since you have 256G RAM
